mmv1/third_party/terraform/services/cloudasset/data_source_google_cloud_asset_resources_search_all.go.erb

```diff
@@ -113,8 +113,14 @@ func datasourceGoogleCloudAssetResourcesSearchAllRead(d *schema.ResourceData, me
 			return err
 		}
 
+		var project string
+    if config.UserProjectOverride && config.BillingProject != "" {
```

Comment:
Could you fix the indentation on the added lines?

Reply:
My bad! Editor must've been configured to use spaces instead of tabs.